vulnerabilities/scraper/ruby.py

```diff
@@ -0,0 +1,104 @@
+import os
+import urllib.request
+from itertools import chain
+from shutil import rmtree
+from urllib.error import HTTPError
+from zipfile import ZipFile
+
+import saneyaml
+from dephell_specifier import RangeSpecifier
+
+RUBYCVE_LINK = 'https://github.com/rubysec/ruby-advisory-db/archive/master.zip'
+DOWNLOAD_PATH = os.path.dirname(os.path.realpath(__file__))
+
+
+def get_rubycve_db():
+    path_to_zip, _ = urllib.request.urlretrieve(
+        RUBYCVE_LINK, os.path.join(
+            DOWNLOAD_PATH, 'ruby.zip'))
+    ZipFile(path_to_zip).extractall(DOWNLOAD_PATH)
+    os.remove(path_to_zip)
+
+
+def path_of_yaml_of_all_packages():
+    gem_path = os.path.join(DOWNLOAD_PATH, 'ruby-advisory-db-master', 'gems')
+    rubies_path = os.path.join(
```

Comment:
See [my previous comment](https://github.com/nexB/vulnerablecode/pull/143/files#diff-b585e6dc3a5cc26d34cbf5ca4b03dcd5R216) about this: I think for now we want to skip the "rubies" directory.